libc/config/linux/x86_64/exclude.txt

```diff
@@ -0,0 +1,16 @@
+# This optional file is used to exclude entrypoints/headers for specific targets.
+
+try_compile(
+  has_sys_random
+  ${CMAKE_CURRENT_BINARY_DIR}
+  SOURCES ${LIBC_SOURCE_DIR}/src/sys/random/linux/getrandom.cpp
+)
```

Comment:
We should only do this in overlay mode right?  Since we do know whether the header exists or not in full build mode.